rotkehlchen/tests/api/test_assets.py

```diff
@@ -61,7 +65,7 @@ def test_query_owned_assets(
                 "ownedassetsresource",
             ),
         )
-    assert_proper_response(response)
```

Comment:
Hmm I think you misunderstood the reason for this change.

`assert_proper_response_with_result` is exactly like `assert_proper_response()` but also gets the response's result. So this PR should reduce overall test lines. Not increase them.

What `assert_proper_response_with_result()` does internally is to:

1. get the response.json()
2. Check `response_data['message'] == ''` -> so that the result is proper
3. Return data['result']

So for example for this particular instance what you should have done is  replace lines 64-67 with:

```python
result = assert_proper_response_with_result(response)
assert(set(result)) == {'ETH', 'BTC', 'EUR', A_RDN.identifier}
```

Reply:
Thanks for the review! I'll make the changes and resubmit.